packages/run-protocol/src/vaultFactory/AttackersGuide.md

```diff
@@ -0,0 +1,75 @@
+# An Attacker's guide to the Vault Factory
+
+This is an incomplete list of potential weak points that an attacker might want to focus
+on when looking for ways to violate the integrity of the Vault Factory. It's here to help
+defenders, as "attacker's mindset" is a good way to focus attention for the defender. The
+list should correspond pretty closely to the set of assurances that the Vault Factory aims
+to support.
+
+The VaultFactory's purpose is to make it possible for users to deposit hard currencies and
+borrow RUN in exchange. They will pay interest on the total outstanding balance, and have
+to pay off the debt or deposit additional collateral to ensure that the ratio of debt to
+collateral stays above a threshold. If the ratio falls below the limit, their collateral
+will be sold.
+
+Part of the purpose of the Vault mechanism is to link the price of RUN to
+dollar-equivalents. We're currently getting prices from the PriceAuthority in RUN, but
+this should (?) be changed to USDC or equivalent. Tying the price at which Vaults lend RUN
+to the USDC price of the collateral is intended to set a ceiling on what one might pay for
+RUN. The fact that vaults get liquidated to the AMM if the USDC-value of the collateral
+falls below the required ratio should restore the value of RUN when collateral values
+fall. In DAI, the same linkage is enforced by auctioning off the collateral. Does selling
+into the AMM achieve the same end?
+
+## VaultManager
+
+Collateral from all borrowers who have deposited the same currency is intermingled. If we
+don't keep good records or can be confused about how much belongs to whom, there could be
+unintended interactions between the vaults held by a single VaultManager on behalf of
+different users.
+
+The VaultManagers calculate interest once daily, and record the change centrally without
+updating individual vaults. Vault owners shouldn't be able to adjust balances or close
```

Comment:
```suggestion
updating individual vaults. Vault owners must not be able to adjust balances or close
```

Reply:
Um, yeah. You've spent more time in RFC-land than I have, but I do know the difference.